### PR TITLE
Will and Cedric Dream Team (Added arm commands)

### PR DESCRIPTION
**Pull request by MartensCedric**
# Assignee Section

## Description
All Arm commands are up to date

closes #42069

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.

- [ ] Local Test Performed Successfully
- [ ] [General Integration Test](https://docs.google.com/document/d/1ug0CpA1cIzURP8DDFSvCt2CEJJSwJ6Ta6B1LG_hYk6I/edit) Performed Successfully

For Pull Requests that do not include code changes, it is not required to perform the tests above.

### CODE REVIEW

File: robot/rospackages/src/mcu_control/scripts/CommsNode.py [hunk 1]
```diff
@@ -36,10 +37,23 @@ def get_handler(commandId, selectedDevice):
...
+    # - check for any new messages to send to teensies
+    # - prompt and receive messages from teensies
+    # - will try to balance the load between sending/receiving so that teensies don't get too much data at once.
 
-def listen_arm():
+    receive_message() # tim pls let me test
+
+def receive_message():
     try:
         while not rospy.is_shutdown():
```
Comment (VimFreeman) [inline robot/rospackages/src/mcu_control/scripts/CommsNode.py:58]: Should be removed no?

**Review by VimFreeman — COMMENTED** [review]
Looks good

File: robot/rover/Arm/Arm.ino [hunk 2]
```diff
@@ -823,13 +823,19 @@ void setOpenLoopState(uint8_t motorId, bool isOpenLoop)
...
 
-void budgeMotors(uint8_t* motorsToMove, bool* moveCW)
+#define MOVE_MOTOR_IDLE 0
+#define MOVE_MOTOR_FORWARD 1
+#define MOVE_MOTOR_BACKWARD 2
+
+void budgeMotors(uint8_t motorActions[])
 {
-    while(motorsToMove != 0)
+    for(uint8_t i = 0; i < 6; i++)
```
Comment (ginywiny) [inline robot/rover/Arm/Arm.ino:832]: Would it be better to use NUM_MOTORS instead of the value 6?

File: robot/rover/Arm/Arm.ino [hunk 3]
```diff
@@ -846,15 +852,13 @@ void resetSingleMotor(uint8_t motorId)
     motorArray[motorId - 1]->setSoftwareAngle(0.0);
 }
 
-void moveMultipleMotors(uint8_t* motorsToMove, float* anglesToReach)
+void moveMultipleMotors(float anglesToReach[])
 {
-    while(*motorsToMove != 0)
+    for(uint8_t i = 0; i < 6; i++)
```
Comment (ginywiny) [inline robot/rover/Arm/Arm.ino:857]: Over here as well?

**Comment by MartensCedric** [comment]
yes, good idea

**Review by ginywiny — COMMENTED** [review]
looking NICE, just nitpicking some variables, but it looks good!

**Pull request merged**